addons/modules/stringtable.xml

```diff
@@ -30,6 +33,7 @@
         </Key>
         <Key ID="STR_CBA_Modules_AttackLocType">
             <English>Attack Location Type</English>
+            <German>?????</German>
```

Comment:
That doesn't seem correct. Also very hard to pronounce.
"Fragenzeichen Fragezeichen Fragezeichen Fragezeichen Fragezeichen"
Seems very long.

Reply:
Placeholder, because WIP, and it may already be in the game.